common/agents/tasks/scripts/forge-review-append-comment.sh

```diff
@@ -0,0 +1,160 @@
+#
+# Arguments:
+#   --file PATH        File path relative to repo root (e.g., src/lib.rs)
+#   --line NUM         Line number in the file
+#   --match TEXT       Text that must appear on that line (for validation)
+#   --body COMMENT     The review comment body
+#   --review-file FILE Output JSONL file (default: .git/review.jsonl)
+#   --old-path PATH    Original file path for renames (GitLab only)
+#
+# The review file must first be created with forge-review-start.sh.
+#
+# The script verifies that --match text appears on the specified line before
+# appending. This prevents comments from being attached to wrong lines when
+# the file has changed.
+#
+# Example:
+#   forge-review-append-comment.sh \
+#     --file src/lib.rs \
+#     --line 42 \
+#     --match "fn process_data" \
+#     --body "AI: *Important*: Missing error handling for empty input"
+
+set -euo pipefail
+
+usage() {
+    cat >&2 <<EOF
+Usage: $0 --file PATH --line NUM --match TEXT --body COMMENT [OPTIONS]
+
+Required:
+  --file PATH        File path relative to repo root
+  --line NUM         Line number in the file
+  --match TEXT       Text that must appear on that line (for validation)
+  --body COMMENT     The review comment body
+
+Optional:
+  --review-file FILE Output JSONL file (default: .git/review.jsonl)
+  --old-path PATH    Original file path for renames (GitLab compatibility)
+
+Example:
+  $0 --file src/lib.rs --line 42 --match "fn process" --body "AI: Add docs"
+EOF
+    exit 1
+}
+
+FILE=""
+LINE=""
+MATCH=""
+BODY=""
+REVIEW_FILE=""
+OLD_PATH=""
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --file)
+            FILE="$2"
+            shift 2
+            ;;
+        --line)
+            LINE="$2"
+            shift 2
+            ;;
+        --match)
+            MATCH="$2"
+            shift 2
+            ;;
+        --body)
+            BODY="$2"
+            shift 2
+            ;;
+        --review-file)
+            REVIEW_FILE="$2"
+            shift 2
+            ;;
+        --old-path)
+            OLD_PATH="$2"
+            shift 2
+            ;;
+        -h|--help)
+            usage
+            ;;
+        *)
+            echo "Error: Unknown option: $1" >&2
+            usage
+            ;;
+    esac
+done
+
+# Validate required arguments
+if [[ -z "$FILE" ]]; then
+    echo "Error: --file is required" >&2
+    usage
+fi
+if [[ -z "$LINE" ]]; then
+    echo "Error: --line is required" >&2
+    usage
+fi
+if [[ -z "$MATCH" ]]; then
+    echo "Error: --match is required" >&2
+    usage
+fi
+if [[ -z "$BODY" ]]; then
+    echo "Error: --body is required" >&2
+    usage
+fi
+
+# Validate line is a number
+if ! [[ "$LINE" =~ ^[0-9]+$ ]]; then
+    echo "Error: --line must be a positive integer, got: $LINE" >&2
+    exit 1
+fi
+
+# Check file exists
+if [[ ! -f "$FILE" ]]; then
+    echo "Error: File not found: $FILE" >&2
+    exit 1
+fi
+
+# Extract the actual line content
+ACTUAL_LINE=$(sed -n "${LINE}p" "$FILE")
```

Comment:
AI: (low) Using `sed` to extract line content works, but per REVIEW.md guidelines about structured data, consider using a tool that provides better error handling. That said, for plain text line extraction this is reasonable.